gnome-extra/chrome-gnome-shell/chrome-gnome-shell-10-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="GNOME Shell integration for Chrome/Chromium, Firefox, Vivaldi, Opera browsers"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeShellIntegrationForChrome"
+SRC_URI="mirror://gnome/sources/${PN}/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ia64 ~ppc ~ppc64 ~x86"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}
+	app-misc/jq
+	sys-apps/coreutils
+"
+RDEPEND="${PYTHON_DEPS}
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	gnome-base/gnome-shell
+"
+
+src_configure() {
+	local mycmakeargs=( -DBUILD_EXTENSION=OFF )
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	if [[ $(get_libdir) != "lib" && "${SYMLINK_LIB}" != yes ]]; then
```

Comment:
Only question I have is "are we allowed to rely on SYMLINK_LIB value to do this ?"

Reply:
Also, isn't /lib or /usr/lib supposed to be the native bitness supported by the arch always ? I know systemd installs a lot of files directly to /usr/lib

Reply:
> Only question I have is "are we allowed to rely on SYMLINK_LIB value to do this ?"

There are ebuilds exists that using SYMLINK_LIB in the tree.

> Also, isn't /lib or /usr/lib supposed to be the native bitness supported by the arch always 

No according to https://gitweb.gentoo.org/data/gentoo-news.git/plain/2017-12-26-experimental-amd64-17-1-profiles/2017-12-26-experimental-amd64-17-1-profiles.en.txt